### ISSUE TITLE
Does not work with Elasticsearch 7.6.2

### ISSUE THREAD

**Issue by Stalso** [deepstreamIO/deepstream.io-storage-elasticsearch#9]
If you try to create record using this connector and **Elasticsearch 7.6.2** you will get elastic search internal error. Everything is ok with **Elasticsearch 6.4.0**. I think, the reason is in new  **Elasticsearch 7.x** API. Also issue may be caused by the fact, that [elasticsearch.js](https://www.npmjs.com/package/elasticsearch) you use is deprecated

**Comment by yasserf** [comment]
Thanks for raising. Are you okay with raising a PR by any chance?

**Comment by Stalso** [comment]
@yasserf I can provide you with some initial implementation, but it will definetely not be ideal. I am .NET developer, so my js knowledge is rather limited.

**Comment by yasserf** [comment]
@valentinvichnal is this something you can look at maybe?

**Comment by valentinvichnal** [comment]
Yes, I will try to fix it and update to the latest elasticsearch version.

**Comment by yasserf** [comment]
You just need to run run [ncu](https://www.npmjs.com/package/npm-check-updates) to update the deps in package.json and ensure the docker image it runs against is the latest version.
